examples/prod-cluster-with-three-workers.yaml

```diff
@@ -19,9 +19,11 @@ metadata:
 spec:
   runtimeVersions:
     sparkVersion: "4.0.0-preview1"
-  initWorkers: 3
-  minWorkers: 3
-  maxWorkers: 3
+  clusterTolerations:
```

Comment:
BTW, this is irrelevant to the user provided spec. Please proceed this separately because this means that we need observers.  That's the real reason why I decided to exclude this part initially.